.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -57,7 +57,7 @@ run_sanity_check() {
         "ray[cpp]==$RAY_VERSION"
     (
         cd release/util
-        python sanity_check.py
```

Comment:
why is this `BUILDKTIE_COMMIT` but the linux one `RAY_HASH`?

Reply:
the linux script is used for other places other than Buildkite (e.g. calling `pip_download_test.sh`) so calling it `BUILDKITE_COMMIT` doesn't make sense here I think. That step `export RAY_HASH=BUILDKITE_COMMIT` so it's still the same `BUILDKITE_COMMIT` when run on buildkite, but user can modify the hash when they run the script manually.

Reply:
fixed

Reply:
it's now reading `BUILDKITE_COMMIT` straight from the script without exporting